src/components/customSearchbox/SearchBoxView.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Why do we need this global rule and not in the specific code block?

Reply:
there were some purposefully unused destructured variables, but I've added the rule to each of them instead.
in 814765a